functions.php

```diff
@@ -96,6 +96,14 @@ function twentynineteen_setup() {
 endif;
 add_action( 'after_setup_theme', 'twentynineteen_setup' );
 
+
+/**
+ * Check if AMP plugin is enabled and if AMP endpoint.
+ */
+function twentynineteen_amp_enabled() {
```

Comment:
In WP Rig this would be called `twentynineteen_is_amp()`. I think that is better than `...amp_enabled` because your site may have AMP enabled even though it is not being served at the moment, as in the case of paired mode or if you opt-out of AMP for a given post. 

https://github.com/wprig/wprig/blob/d8d9cca76027b3057d4df2ce7cc3bd3101172281/dev/inc/template-tags.php#L10-L20